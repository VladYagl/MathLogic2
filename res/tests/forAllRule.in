Phi->Psi(x), Betta, Alpha|-Phi->@xPsi(x)
Phi->Psi(x)
Phi->@xPsi(x)