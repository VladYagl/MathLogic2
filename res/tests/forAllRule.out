(Phi)->(Psi(x)),Betta|-(Alpha)->((Phi)->(@x(Psi(x))))
(Phi)->(Psi(x))
((Phi)->(Psi(x)))->((Alpha)->((Phi)->(Psi(x))))
(Alpha)->((Phi)->(Psi(x)))
((Alpha)&(Phi))->(((Alpha)&(Phi))->((Alpha)&(Phi)))
((Alpha)&(Phi))->((((Alpha)&(Phi))->((Alpha)&(Phi)))->((Alpha)&(Phi)))
(((Alpha)&(Phi))->(((Alpha)&(Phi))->((Alpha)&(Phi))))->((((Alpha)&(Phi))->((((Alpha)&(Phi))->((Alpha)&(Phi)))->((Alpha)&(Phi))))->(((Alpha)&(Phi))->((Alpha)&(Phi))))
(((Alpha)&(Phi))->((((Alpha)&(Phi))->((Alpha)&(Phi)))->((Alpha)&(Phi))))->(((Alpha)&(Phi))->((Alpha)&(Phi)))
((Alpha)&(Phi))->((Alpha)&(Phi))
((Alpha)&(Phi))->(Alpha)
(((Alpha)&(Phi))->(Alpha))->(((Alpha)&(Phi))->(((Alpha)&(Phi))->(Alpha)))
((Alpha)&(Phi))->(((Alpha)&(Phi))->(Alpha))
((Alpha)&(Phi))->(Phi)
(((Alpha)&(Phi))->(Phi))->(((Alpha)&(Phi))->(((Alpha)&(Phi))->(Phi)))
((Alpha)&(Phi))->(((Alpha)&(Phi))->(Phi))
(((Alpha)&(Phi))->((Alpha)&(Phi)))->((((Alpha)&(Phi))->(((Alpha)&(Phi))->(Alpha)))->(((Alpha)&(Phi))->(Alpha)))
(((Alpha)&(Phi))->(((Alpha)&(Phi))->(Alpha)))->(((Alpha)&(Phi))->(Alpha))
((Alpha)&(Phi))->(Alpha)
(((Alpha)&(Phi))->((Alpha)&(Phi)))->((((Alpha)&(Phi))->(((Alpha)&(Phi))->(Phi)))->(((Alpha)&(Phi))->(Phi)))
(((Alpha)&(Phi))->(((Alpha)&(Phi))->(Phi)))->(((Alpha)&(Phi))->(Phi))
((Alpha)&(Phi))->(Phi)
(Alpha)->((Phi)->(Psi(x)))
((Alpha)->((Phi)->(Psi(x))))->(((Alpha)&(Phi))->((Alpha)->((Phi)->(Psi(x)))))
((Alpha)&(Phi))->((Alpha)->((Phi)->(Psi(x))))
(((Alpha)&(Phi))->(Alpha))->((((Alpha)&(Phi))->((Alpha)->((Phi)->(Psi(x)))))->(((Alpha)&(Phi))->((Phi)->(Psi(x)))))
(((Alpha)&(Phi))->((Alpha)->((Phi)->(Psi(x)))))->(((Alpha)&(Phi))->((Phi)->(Psi(x))))
((Alpha)&(Phi))->((Phi)->(Psi(x)))
(((Alpha)&(Phi))->(Phi))->((((Alpha)&(Phi))->((Phi)->(Psi(x))))->(((Alpha)&(Phi))->(Psi(x))))
(((Alpha)&(Phi))->((Phi)->(Psi(x))))->(((Alpha)&(Phi))->(Psi(x)))
((Alpha)&(Phi))->(Psi(x))
((Alpha)&(Phi))->(@x(Psi(x)))
(Alpha)->((Alpha)->(Alpha))
(Alpha)->(((Alpha)->(Alpha))->(Alpha))
((Alpha)->((Alpha)->(Alpha)))->(((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha)))
((Alpha)->(((Alpha)->(Alpha))->(Alpha)))->((Alpha)->(Alpha))
(Alpha)->(Alpha)
(Alpha)->((Phi)->(Alpha))
((Alpha)->((Phi)->(Alpha)))->((Alpha)->((Alpha)->((Phi)->(Alpha))))
(Alpha)->((Alpha)->((Phi)->(Alpha)))
((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Phi)->(Alpha))))->((Alpha)->((Phi)->(Alpha))))
((Alpha)->((Alpha)->((Phi)->(Alpha))))->((Alpha)->((Phi)->(Alpha)))
(Alpha)->((Phi)->(Alpha))
(Phi)->((Phi)->(Phi))
((Phi)->((Phi)->(Phi)))->((Alpha)->((Phi)->((Phi)->(Phi))))
(Alpha)->((Phi)->((Phi)->(Phi)))
(Phi)->(((Phi)->(Phi))->(Phi))
((Phi)->(((Phi)->(Phi))->(Phi)))->((Alpha)->((Phi)->(((Phi)->(Phi))->(Phi))))
(Alpha)->((Phi)->(((Phi)->(Phi))->(Phi)))
((Phi)->((Phi)->(Phi)))->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi)))
(((Phi)->((Phi)->(Phi)))->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi))))->((Alpha)->(((Phi)->((Phi)->(Phi)))->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi)))))
(Alpha)->(((Phi)->((Phi)->(Phi)))->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi))))
((Alpha)->((Phi)->((Phi)->(Phi))))->(((Alpha)->(((Phi)->((Phi)->(Phi)))->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi)))))->((Alpha)->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi)))))
((Alpha)->(((Phi)->((Phi)->(Phi)))->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi)))))->((Alpha)->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi))))
(Alpha)->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi)))
((Alpha)->((Phi)->(((Phi)->(Phi))->(Phi))))->(((Alpha)->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi))))->((Alpha)->((Phi)->(Phi))))
((Alpha)->(((Phi)->(((Phi)->(Phi))->(Phi)))->((Phi)->(Phi))))->((Alpha)->((Phi)->(Phi)))
(Alpha)->((Phi)->(Phi))
(Alpha)->((Phi)->((Alpha)&(Phi)))
((Alpha)->((Phi)->((Alpha)&(Phi))))->((Alpha)->((Alpha)->((Phi)->((Alpha)&(Phi)))))
(Alpha)->((Alpha)->((Phi)->((Alpha)&(Phi))))
((Alpha)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))
(((Alpha)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi))))))->((Alpha)->(((Alpha)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))))
(Alpha)->(((Alpha)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi))))))
((Alpha)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->(((Alpha)->(((Alpha)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))))->((Alpha)->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))))
((Alpha)->(((Alpha)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))))->((Alpha)->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi))))))
(Alpha)->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))
((Phi)->(Alpha))->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi)))))
(((Phi)->(Alpha))->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi))))))->((Alpha)->(((Phi)->(Alpha))->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi)))))))
(Alpha)->(((Phi)->(Alpha))->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi))))))
((Alpha)->((Phi)->(Alpha)))->(((Alpha)->(((Phi)->(Alpha))->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi)))))))->((Alpha)->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi)))))))
((Alpha)->(((Phi)->(Alpha))->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi)))))))->((Alpha)->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi))))))
(Alpha)->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi)))))
((Alpha)->((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi))))))->(((Alpha)->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi))))))->((Alpha)->((Phi)->((Phi)->((Alpha)&(Phi))))))
((Alpha)->(((Phi)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Phi)->((Phi)->((Alpha)&(Phi))))))->((Alpha)->((Phi)->((Phi)->((Alpha)&(Phi)))))
(Alpha)->((Phi)->((Phi)->((Alpha)&(Phi))))
((Phi)->(Phi))->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi))))
(((Phi)->(Phi))->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi)))))->((Alpha)->(((Phi)->(Phi))->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi))))))
(Alpha)->(((Phi)->(Phi))->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi)))))
((Alpha)->((Phi)->(Phi)))->(((Alpha)->(((Phi)->(Phi))->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi))))))->((Alpha)->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi))))))
((Alpha)->(((Phi)->(Phi))->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi))))))->((Alpha)->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi)))))
(Alpha)->(((Phi)->((Phi)->((Alpha)&(Phi))))->((Phi)->((Alpha)&(Phi))))
((Alpha)->(Alpha))->(((Alpha)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Alpha)->((Phi)->((Alpha)&(Phi)))))
((Alpha)->((Alpha)->((Phi)->((Alpha)&(Phi)))))->((Alpha)->((Phi)->((Alpha)&(Phi))))
(Alpha)->((Phi)->((Alpha)&(Phi)))
((Alpha)&(Phi))->(@x(Psi(x)))
(((Alpha)&(Phi))->(@x(Psi(x))))->((Alpha)->(((Alpha)&(Phi))->(@x(Psi(x)))))
(Alpha)->(((Alpha)&(Phi))->(@x(Psi(x))))
(((Alpha)&(Phi))->(@x(Psi(x))))->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))
((((Alpha)&(Phi))->(@x(Psi(x))))->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x))))))->((Alpha)->((((Alpha)&(Phi))->(@x(Psi(x))))->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))))
(Alpha)->((((Alpha)&(Phi))->(@x(Psi(x))))->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x))))))
((Alpha)->(((Alpha)&(Phi))->(@x(Psi(x)))))->(((Alpha)->((((Alpha)&(Phi))->(@x(Psi(x))))->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))))->((Alpha)->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))))
((Alpha)->((((Alpha)&(Phi))->(@x(Psi(x))))->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))))->((Alpha)->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x))))))
(Alpha)->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))
((Phi)->((Alpha)&(Phi)))->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x)))))
(((Phi)->((Alpha)&(Phi)))->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x))))))->((Alpha)->(((Phi)->((Alpha)&(Phi)))->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x)))))))
(Alpha)->(((Phi)->((Alpha)&(Phi)))->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x))))))
((Alpha)->((Phi)->((Alpha)&(Phi))))->(((Alpha)->(((Phi)->((Alpha)&(Phi)))->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x)))))))->((Alpha)->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x)))))))
((Alpha)->(((Phi)->((Alpha)&(Phi)))->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x)))))))->((Alpha)->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x))))))
(Alpha)->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x)))))
((Alpha)->((Phi)->(((Alpha)&(Phi))->(@x(Psi(x))))))->(((Alpha)->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x))))))->((Alpha)->((Phi)->(@x(Psi(x))))))
((Alpha)->(((Phi)->(((Alpha)&(Phi))->(@x(Psi(x)))))->((Phi)->(@x(Psi(x))))))->((Alpha)->((Phi)->(@x(Psi(x)))))
(Alpha)->((Phi)->(@x(Psi(x))))
